### PR TITLE
BUG Fix missing dist images

### LINKED ISSUE #953: Missing file icon prevents video embed
## Affected Version

A freshly generated CWP demo, running SilverStripe (Version - CMS: 4.3.3, CWP: 2.2.3)

## Description

I'm unable to use the WYSIWYG editor to embed a Facebook video because a PHP notice about a missing file is displayed in the Network response. 

## Steps to Reproduce

1. Generate a new CWP demo
2. Find some random Facebook videos, maybe any of these ones:
    a) https://www.facebook.com/brutnature/videos/the-honey-badger-is-not-afraid-of-anything/220639575159714/
    b) https://www.facebook.com/renewsnz/videos/185151292018376/
3. Click the insert media button and enter the URL.
4. Observe the following error:
```
Unexpected token < in JSON at position 0
```
5. This is in the network response, just before the JSON schema:
```
<br />
<b>Notice</b>:  File resources/vendor/silverstripe/asset-admin/client/dist/images/icon_file.png does not exist in <b>/var/www/html/vendor/silverstripe/framework/src/Control/SimpleResourceURLGenerator.php</b> on line <b>90</b><br />
<br />
<b>Notice</b>:  File resources/vendor/silverstripe/asset-admin/client/dist/images/icon_file.png does not exist in <b>/var/www/html/vendor/silverstripe/framework/src/Control/SimpleResourceURLGenerator.php</b> on line <b>90</b><br />
```

### PR DESCRIPTION
**Pull request by tractorcow**
Fixes #953

Just copied these from src/images to dist/images fixes the issue.

### CODE REVIEW

**Comment by maxime-rainville** [comment]
Just pushed an update to auto copy the `client/src/images` directory to `client/dist/images` on every build.

File: .travis.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ before_script:
   - if [[ $BEHAT_TEST ]]; then (vendor/bin/serve --bootstrap-file vendor/silverstripe/cms/tests/behat/serve-bootstrap.php &> artifacts/serve.log &); fi
 
 # Install NPM dependencies
-  - if [[ $NPM_TEST ]]; then nvm install && nvm use && npm install -g yarn && yarn install --network-concurrency 1 && (cd vendor/silverstripe/admin && yarn install --network-concurrency 1) && yarn run build; fi
+  - if [[ $NPM_TEST ]]; then rm -rf client/dist && nvm install && nvm use && npm install -g yarn && yarn install --network-concurrency 1 && (cd vendor/silverstripe/admin && yarn install --network-concurrency 1) && yarn run build; fi
```
Comment (maxime-rainville) [inline .travis.yml]: By completely deleting the `client/dist` directory before the build, we make sure the build completely rebuild the dist folder.

File: package.json [hunk 2]
```diff
@@ -54,6 +54,7 @@
     "@storybook/addons": "^3.4.11",
     "@storybook/react": "^3.4.11",
     "babel-jest": "^23.6.0",
+    "copy-webpack-plugin": "^4",
```
Comment (maxime-rainville) [inline package.json]: The latest version of "copy-webpack-plugin" is 5, but it's not compatible with webpack 2.

**Comment by NightJar** [comment]
Thanks fellows, nice work both of you :)

**Pull request merged**